setup.cfg

```diff
@@ -35,6 +35,7 @@ install_requires =
     google-auth >= 2.6
     fontawesomefree >= 6.1
     django-autocomplete-light >= 3.9
+    django-filter > 2.0
```

Comment:
Don't think this version is correct

Reply:
@wkirdp Can you look into this? Did you mean > 23.0? >=23.0?

Reply:
@amstilp I pinned it to 23.2 and you said maybe not to pin to such a high version. Maybe just >2.0.  Did I misunderstand?  (After version 2.4, it jumped to 21.1)

Reply:
Sorry, I saw version 23.2 in requirements/base.txt and >2 in setup.cfg and thought it was unintentional. Ignore if that version makes sense.